### PR TITLE
Add Chinese Translation for 0x00 0x02 0x03

### PR DESCRIPTION
**Pull request by SpZanG**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [x] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [x] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [x] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue  #340 

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: Document-zhcn/0x02-Frontispiece.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+MASVS是社区努力和行业反馈的成果。我们希望该标准会随着时间的推移不断更新，并且非常欢迎来自社区的意见反馈。
+
+与我们取得联系的最佳方式是通过 OWASP Mobile Project Slack频道: <https://owasp.slack.com/messages/project-mobile_omtg/details/>
+
+可以通过以下URL创建帐户: [https://owasp.slack.com/](https://join.slack.com/t/owasp/shared_invite/enQtNjExMTc3MTg0MzU4LWQ2Nzg3NGJiZGQ2MjRmNzkzN2Q4YzU1MWYyZTdjYjA2ZTA5M2RkNzE2ZjdkNzI5ZThhOWY5MjljYWZmYmY4ZjM)
+
+
+## 版权和许可
+
+![license](images/CC-license.png) 版权所有©2018 OWASP基金会。本作品是根据[知识共享署名-相同方式共享 4.0 国际许可](https://creativecommons.org/licenses/by-sa/4.0/). 对于任何重复使用或分发，您必须向他人明确此作品的许可条款.
```
Comment (boblone19) [inline Document-zhcn/0x02-Frontispiece.md]: license 版权所有© 2019 OWASP基金会。本作品是根据[知识共享署名-相同方式共享 4.0 国际许可](https://creativecommons.org/licenses/by-sa/4.0/). 对于任何重复使用或分发，您必须向他人明确此作品的许可条款.

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# 移动应用安全验证标准
+
+MASVS 可以被用于确立移动应用程序的安全程度。此标准(MASVS)的建立遵循了下面三个主要目标：
```
Comment (boblone19) [inline Document-zhcn/0x03-Using_the_MASVS.md]: MASVS 可以用来对移动应用程序安全性信心的建立. 制定这些要求时,需要考虑到以下目标:

File: Document-zhcn/0x02-Frontispiece.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+# 移动应用安全验证标准
+
+## 关于标准
+
+欢迎阅读移动应用安全验证标准（MASVS）1.1版. MASVS 是一个社区项目，其宗旨是建立一个安全需求的框架给人们去设计、开发和测试iOS和Android的移动应用程序。
+
+MASVS是社区努力和行业反馈的成果。我们希望该标准会随着时间的推移不断更新，并且非常欢迎来自社区的意见反馈。
+
+与我们取得联系的最佳方式是通过 OWASP Mobile Project Slack频道: <https://owasp.slack.com/messages/project-mobile_omtg/details/>
```
Comment (jack-smith-dh) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
与我们取得联系的最佳方式是通过OWASP Mobile Project Slack频道: <https://owasp.slack.com/messages/project-mobile_omtg/details/>
```

File: Document-zhcn/0x02-Frontispiece.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+| 西班牙语 | Martin Marsicano, Carlos Holguera |
+| 日语 | Koki Takeyama, Riotaro Okada (审核) |
+| 俄语 | Gall Maxim, Chelnokov Vladislav (审核), Oprya Egor (审核), Tereshin Dmitry (审核) |
+| 韩语 | Youngjae Jeon, Jeongwon Cho, Jiyou Han, Jiyeon Sung |
+
+本文档是由Jim Manico编写的OWASP Application Security Verification Standard的一个分支。
+
+## 赞助商
+
+虽然MASVS和MSTG都是由社区自愿创建和维护的，但是有时候仍然需要一点外部的帮助。因此，我们感谢我们的赞助商提供资金来聘请技术编辑。请注意，他们的赞助不会以任何方式影响MASVS或MSTG的内容。赞助方案的具体细节：[OWASP项目Wiki](https://www.owasp.org/index.php/OWASP_Mobile_Security_Testing_Guide#tab=Sponsorship_Packages "OWASP Mobile Security Testing Guide Sponsorship Packages").
```
Comment (jack-smith-dh) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
虽然MASVS和MSTG都是由社区自愿创建和维护的，但是有时候仍然需要一点外部的帮助。因此，我们感谢我们的赞助商提供资金来聘请技术编辑。但是，他们的赞助不会以任何方式影响MASVS或MSTG的内容。赞助方案的具体细节：[OWASP项目Wiki](https://www.owasp.org/index.php/OWASP_Mobile_Security_Testing_Guide#tab=Sponsorship_Packages "OWASP Mobile Security Testing Guide Sponsorship Packages").
```

File: Document-zhcn/0x02-Frontispiece.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+### 特别致谢
+
+[![NowSecure](images/NowSecure_logo.png)](https://www.nowsecure.com/ "NowSecure")
+
+### 好心人赞助
+
+[![RandoriSec](images/Randorisec_logo.png)](https://www.randorisec.fr/ "RandoriSec")
+
+接下来，我们要感谢OWASP海湾地区分会的赞助。最后，我们要感谢每一位从Leanpub购买了这本书并以这种方式赞助我们的人。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
接下来，我们要感谢OWASP湾区分会的赞助。最后，我们要感谢每一位从Leanpub购买了这本书并以这种方式赞助我们的人。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+- 可用于业界应用的安全度量标准 - 提供一个安全标准，让开发者和使用者们可以比较现有的移动应用程序；
+- 可用于指导纲领 - 为移动应用程序整个开发阶段和测试提供安全指引；
+- 可用于采购检验的验证参考 - 提供一个移动应用程序安全的验证标准。
+
+## 移动应用安全模型 (Mobile AppSec Model)
+
+MASVS定义了两个安全验证级别（MASVS-L1 和 MASVS-L2），以及一系列的抵抗逆向工程防御韧性准则（MASVS-R）。 MASVS-L1 是建议所有的移动应用程序都应该遵守的要求，MASVS-L2 则是针对处理高度敏感数据的移动应用程序。MASVS-R提供了需要防范客户端威胁时，可以使用的额外防护管控方法。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
MASVS定义了两个安全验证级别（MASVS-L1 和 MASVS-L2），以及一系列的抵抗逆向工程防御韧性准则（MASVS-R）。 MASVS-L1是建议所有的移动应用程序都应该遵守的要求，MASVS-L2则针对的是处理高度敏感数据的移动应用程序。MASVS-R提供了需要防范客户端威胁时可以使用的额外防护管控方法。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+满足 MASVS-L1 中所提到的要求，可以确保移动应用程序的安全，使开发流程符合信息安全的最佳实践方法，并且确保移动应用程序可以防御常见的安全问题。 MASVS-L2添加了额外的纵深防御（defense-in-depth），例如凭证绑定（SSL pinning）。因此，MASVS-L2可以使移动应用程序能够更好的防范复杂的攻击行为 - 假设移动操作系统的安全控件完好无损，并且最终用户不被视为潜在的恶意用户。满足全部或者部分的MASVS-R软件保护要求，可以帮助阻止特定的客户端威胁，包括最终用户是恶意用户和手机操作系统遭到破坏。
+
+**I：虽然我们建议应该在每一个移动应用程序中实施MASVS-L1准则，但是否实施最终应是基于风险的决策，该决策应与企业所有者沟通。**
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
**I：虽然我们建议应该在每一个移动应用程序中实施MASVS-L1准则，但是否实施最终应是基于风险的决策，该决策应与软件所有者沟通。**
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+- 可用于业界应用的安全度量标准 - 提供一个安全标准，让开发者和使用者们可以比较现有的移动应用程序；
+- 可用于指导纲领 - 为移动应用程序整个开发阶段和测试提供安全指引；
+- 可用于采购检验的验证参考 - 提供一个移动应用程序安全的验证标准。
+
+## 移动应用安全模型 (Mobile AppSec Model)
+
+MASVS定义了两个安全验证级别（MASVS-L1 和 MASVS-L2），以及一系列的抵抗逆向工程防御韧性准则（MASVS-R）。 MASVS-L1 是建议所有的移动应用程序都应该遵守的要求，MASVS-L2 则是针对处理高度敏感数据的移动应用程序。MASVS-R提供了需要防范客户端威胁时，可以使用的额外防护管控方法。
+
+满足 MASVS-L1 中所提到的要求，可以确保移动应用程序的安全，使开发流程符合信息安全的最佳实践方法，并且确保移动应用程序可以防御常见的安全问题。 MASVS-L2添加了额外的纵深防御（defense-in-depth），例如凭证绑定（SSL pinning）。因此，MASVS-L2可以使移动应用程序能够更好的防范复杂的攻击行为 - 假设移动操作系统的安全控件完好无损，并且最终用户不被视为潜在的恶意用户。满足全部或者部分的MASVS-R软件保护要求，可以帮助阻止特定的客户端威胁，包括最终用户是恶意用户和手机操作系统遭到破坏。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
满足MASVS-L1中所提到的要求，可以确保移动应用程序的安全，使开发流程符合信息安全的最佳实践方法，并且确保移动应用程序可以防御常见的安全问题。 MASVS-L2添加了额外的纵深防御（defense-in-depth），例如证书绑定（SSL pinning）。因此，MASVS-L2可以使移动应用程序能够更好的防范复杂的攻击行为 - 假设移动操作系统的安全控件完好无损，并且最终用户不被视为潜在的恶意用户。满足全部或者部分的MASVS-R软件保护要求，可以帮助阻止特定的客户端威胁，包括最终用户是恶意用户或者手机操作系统遭到破坏的情况。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+**I：虽然我们建议应该在每一个移动应用程序中实施MASVS-L1准则，但是否实施最终应是基于风险的决策，该决策应与企业所有者沟通。**
+
+**II：请注意，MASVS-R和<OWASP移动安全性测试指南>中所提到的软件保护准则无法确保不被绕过，绝对不能用作安全管控的替代品。相反的，这些软件保护准则旨在增加已落实MASVS-L1或MASVS- L2的移动应用程序，针对特定的威胁提供额外的防御。**
+
+![Verification Levels](images/masvs-levels-new.jpg)
+
+### 文档结构
+
+MASVS的第一部分阐述了移动应用的安全模型和可用的安全验证级别，接着是有关如何使用本标准的建议。第二部分则是阐述了安全准则的细节和和对应的安全验证登记。基于技术的的目的/范畴，此准则被分成8个类别（从V1到V8）。准则类别与条目的命名方法如下（MASVS和MSTG使用相同的方法命名）：
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
MASVS的第一部分阐述了移动应用的安全模型和可用的安全验证级别，接着是有关如何使用本标准的建议。第二部分则是阐述了安全准则的细节和和对应的安全验证级别。基于技术的的目的或范畴，此准则被分成8个类别（从V1到V8）。准则类别与条目的命名方法如下（MASVS和MSTG使用相同的方法命名）：
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+- *准则类别：* MASVS-Vx，例如：MASVS-V2: 数据存储和隐私
+- *准则条目：* MASVS-Vx.y，例如： MASVS-V2.2: "敏感数据不应该被写入应用程序日志"
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
- *准则条目：* MASVS-Vx.y，例如： MASVS-V2.2: "敏感数据不应被写入应用程序日志"
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+
+### 安全验证等级细节
+
+#### MASVS-L1: 标准安全等级
+
+实现MASVS-L1的移动应用程序需要遵循移动应用程序安全的最佳实践方法。它包含了代码的质量，敏感数据处理以及与移动环境交互的基本安全准则。此外，MASVS-L1必须有一个测试流程来验证安全管控。MASVS-L1适用于所有的移动应用程序。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
实现MASVS-L1的移动应用程序需要遵循移动应用程序安全的最佳实践方法。它包含了代码的质量，敏感数据的处理以及与移动环境交互的基本安全准则。此外，MASVS-L1必须有一个测试流程来验证安全管控。MASVS-L1适用于所有的移动应用程序。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+
+#### MASVS-L2: 纵深防御（Defense-in-Depth）
+
+MASVS-L2引入了高于标准要求的高级安全管控方法。为了实现MASVS-L2，首先需要有一个威胁模型，并且把安全视为应用程序的一部分，加入到应用程序的架构和设计之中。MASVS-L2适用于处理高度敏感数据的应用程序，比如：移动银行应用程序。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
MASVS-L2引入了高于标准要求的高级安全管控方法。为了实现MASVS-L2，首先需要有一个威胁模型，并且把安全视为应用程序的一部分，加入到应用程序的架构和设计之中。MASVS-L2适用于处理高度敏感数据的应用程序，例如手机银行应用程序。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+
+#### MASVS-R: 抵抗逆向工程和篡改的能力
+
+符合MASVS-R的应用程序需具有最先进的安全性，并且可以抵抗有明确定义的特定客户端攻击，例如：篡改，程序修改或者逆向工程提取敏感代码或数据。这样的应用程序需要使用硬件安全模块或者是使用足够强大且验证过的软件保护技术。 MASVS-R适用于处理高度敏感数据的应用程序，并且可以用作保护知识产权或防篡改应用程序的一种方式。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
符合MASVS-R的应用程序需具有最先进的安全性，并且可以抵抗有明确定义的特定客户端攻击，例如：篡改，程序修改或者逆向工程提取敏感代码或数据。这样的应用程序需要使用硬件安全模块或者是使用足够强大且经过验证的软件保护技术。MASVS-R适用于处理高度敏感数据的应用程序，并且可以用作保护知识产权或防篡改应用程序的一种方式。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+
+### 使用建议
+
+基于风险评估和应用程序的安全等级需求，应用程序可以被验证为符合MASVS L1或者L2。L1适用于所有移动应用程序，而L2通常建议用于处理更敏感数据或功能的应用程序。 MASVS-R（或其一部分）可以用于验证应用程序抵抗特定威胁的韧性。例如：重新封装或提取敏感数据。*另外*，也可以被用于更严谨的安全验证。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
基于风险评估和应用程序的安全等级需求，应用程序可以被验证为符合MASVS L1或者L2。L1适用于所有移动应用程序，而L2通常建议用于处理更敏感数据或功能的应用程序。 MASVS-R（或其中的一部分）可以用于验证应用程序抵抗特定威胁的韧性。例如：重新封装或提取敏感数据。*另外*，也可以被用于更严谨的安全验证。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+- MASVS-L1
+- MASVS-L1+R
+- MASVS-L2
+- MASVS-L2+R
+
+不同的验证类别组合反映出不同的安全等级和弹性。这样的组合方式是为了让安全具有灵活性，例如：手机移动游戏程序出于可用性的需求，可能不需要加入MASVS-L2的安全控制机制，如：双因子身份验证，但是在防篡改方面会存在强烈的业务需求。
+
+#### 如何选择安全验证等级和类型
+
+实现MASVS L2的安全准则可以增加安全性，但是也会增加开发成本，并有可能降低最终用户的体验(传统的权衡)。一般来说，是否采用L2来开发应用程序，要考虑风险与成本(比如，当机密性或完整性被破坏所造成的潜在损失高于额外的安全管控机制的成本时，就应该使用L2.) 总体来说，在应用MASVS之前，风险评估应该是第一步。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
实现MASVS L2的安全准则可以增加安全性，但是也会增加开发成本，并有可能降低最终用户的体验，这是一种普遍存在的权衡。一般来说，是否采用L2来开发应用程序，要考虑风险与成本(比如，当机密性或完整性被破坏所造成的潜在损失高于额外的安全管控机制的成本时，就应该使用L2)。总体来说，在应用MASVS之前，风险评估应该是第一步。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+
+##### 举例
+
+###### MASVS-L1
+
+- 应用于所有的移动应用程序。MASVS-L1列举了在不过分影响开发成本以及用户体验的最佳安全实践方法。任何不适用于更严格安全等级的移动应用程序应该采用MASVS-L1标准。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
- 应用于所有的移动应用程序。MASVS-L1列举了在不过分影响开发成本以及用户体验下的最佳安全实践方法。任何不适用于更严格安全等级的移动应用程序应该采用MASVS-L1标准。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 17]
```diff
@@ -0,0 +1,83 @@
...
+
+###### MASVS-L2
+
+- 医疗保健行业：存储个人身份信息的移动应用程序，因为这些信息可被用于身份盗窃，诈骗钱财或者其他的欺骗行为，所以应采用MASVS-L2。对于美国的医疗保健行业，需要遵守相关的信息保密法规 - 《健康保险可移植性和责任法案》（HIPAA）中的隐私权，安全性，违约通知规则和患者安全规则。
+
+- 金融行业：可以访问高度敏感信息（例如：信用卡卡号，个人信息或者转账功能）的移动应用程序。这些移动应用程序需要有额外的安全控制措施来防止诈骗。金融业相关的应用程序需确保遵守支付卡行业数据安全标准（PCI DSS），《格莱姆·里奇·布利利法案》和《萨班斯-奥克斯利法案》（SOX）。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
- 金融行业：可以访问和处理高度敏感信息（例如：信用卡卡号，个人信息或者转账功能）的移动应用程序。这些移动应用程序需要有额外的安全控制措施来防止欺诈行为。金融业相关的应用程序需确保遵守支付卡行业数据安全标准（PCI DSS），《格莱姆·里奇·布利利法案》和《萨班斯-奥克斯利法案》（SOX）。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 18]
```diff
@@ -0,0 +1,83 @@
...
+
+###### MASVS L1+R
+
+- 以知识产权（IP）保护为商业目标的移动应用程序。MASVS-R中列举的弹性控制可以用于增加攻击者获取原始源代码，篡改以及破解的难度。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
- 以知识产权（IP）保护为商业目标的移动应用程序。MASVS-R中列举的韧性控制可以用于增加攻击者获取源代码，篡改以及破解的难度。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 19]
```diff
@@ -0,0 +1,83 @@
...
+
+- 游戏行业：游戏应用程序对于防篡改和作弊有着强烈的需求，例如在线竞技游戏。游戏作弊对于在线游戏来说，是一个至关重要的问题，因为大量作弊者的存在，会导致其他玩家的不满，并且最终导致整个游戏的失败。MASVS-R提供了基础的防篡改控件，从而增加游戏作弊者的工作量。
+
+###### MASVS L2+R
+
+- 金融产业：准许用户执行转账操作的移动银行程序，有代码注入和在被入侵设备上运行的风险。在这种情况下，可以使用MASVS-R的安全管控机制了来防止篡改，从而提高恶意程序开发者的门槛。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
- 金融产业：允许用户执行转账操作的手机银行应用程序，存在代码注入和在被不安全设备中运行的风险。在这种情况下，可以使用MASVS-R的安全管控机制了来防止篡改，从而提高恶意程序开发者的门槛。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 20]
```diff
@@ -0,0 +1,83 @@
...
+
+- 所有被设计需要存储敏感数据在移动设备和支持许多不同种类的移动设备和操作系统的程序，建议采用MASVS L2+R。在这种情况下，安全控制机制可以被用来提高纵深防御，从而增加攻击者提取机密数据的难度。
```
Comment (jack-smith-dh) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
- 所有在移动设备中需要存储敏感数据，以及支持不同种类的移动设备和操作系统的移动应用程序，建议采用MASVS L2+R。在这种情况下，安全控制机制可以被用来提高纵深防御，从而增加攻击者提取机密数据的难度。
```

File: Document-zhcn/0x02-Frontispiece.md [hunk 21]
```diff
@@ -0,0 +1,52 @@
...
+MASVS是社区努力和行业反馈的成果。我们希望该标准会随着时间的推移不断更新，并且非常欢迎来自社区的意见反馈。
+
+与我们取得联系的最佳方式是通过OWASP Mobile Project Slack频道: <https://owasp.slack.com/messages/project-mobile_omtg/details/>
+
+可以通过以下URL创建帐户: [https://owasp.slack.com/](https://join.slack.com/t/owasp/shared_invite/enQtNjExMTc3MTg0MzU4LWQ2Nzg3NGJiZGQ2MjRmNzkzN2Q4YzU1MWYyZTdjYjA2ZTA5M2RkNzE2ZjdkNzI5ZThhOWY5MjljYWZmYmY4ZjM)
+
+
+## 版权和许可
+
+![license](images/CC-license.png) 版权所有©2018 OWASP基金会。本作品是根据[知识共享署名-相同方式共享 4.0 国际许可](https://creativecommons.org/licenses/by-sa/4.0/). 对于任何重复使用或分发，您必须向他人明确此作品的许可条款.
```
Comment (SpZanG) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
![license](images/CC-license.png) 版权所有©2019 OWASP基金会。本作品是根据[知识共享署名-相同方式共享 4.0 国际许可](https://creativecommons.org/licenses/by-sa/4.0/)。对于任何重复使用或分发，您必须向他人明确此作品的许可条款。
```

**Comment by jack-smith-dh** [comment]
reviewed, good to proceed

File: Document-zhcn/0x02-Frontispiece.md [hunk 22]
```diff
@@ -0,0 +1,52 @@
...
+
+| 语言 |翻译和审核 |
+| --- | --- |
+| 中文 | Peter Chi, and Lex Chien, Henry Hu, Leo Wang |
+| 法语 | Romuald Szkudlarek, Christian Dong (审核) |
+| 德文 | Rocco Gränitz, Sven Schleier (审核) |
+| 西班牙语 | Martin Marsicano, Carlos Holguera |
+| 日语 | Koki Takeyama, Riotaro Okada (审核) |
+| 俄语 | Gall Maxim, Chelnokov Vladislav (审核), Oprya Egor (审核), Tereshin Dmitry (审核) |
+| 韩语 | Youngjae Jeon, Jeongwon Cho, Jiyou Han, Jiyeon Sung |
```
Comment (sushi2k) [inline Document-zhcn/0x02-Frontispiece.md]: Can you please make a new row with the names of all chinese translators?

Comment (jack-smith-dh) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
| 韩语 | Youngjae Jeon, Jeongwon Cho, Jiyou Han, Jiyeon Sung |
| 简体中文 | Harold Zang, Bob Peng, JackDH |
```

File: Document-zhcn/0x02-Frontispiece.md [hunk 23]
```diff
@@ -0,0 +1,52 @@
...
+
+| 项目负责人 | 主要作者 | 贡献者和审稿人
+| --- | --- | --- |
+| Sven Schleier，Jeroen Willemsen和Carlos Holguera | Bernhard Mueller |亚历山大·安图克（Alexander Antukh），梅谢里亚科夫（Meshyayakov Aleksey），巴切夫斯基·阿特姆（Bachevsky Artem），耶隆·贝克斯（Jeroen Beckers），弗拉迪斯拉夫·切尔诺科夫（Fladislav Chelnokov），本·切尼（Ben Cheney），彼得·奇（Lex Chien），史蒂芬·科比奥克斯（Stephen Corbiaux），曼努埃尔·德尔加多（Manuel Delgado），拉切科·丹尼斯（Ratchenko Denis），瑞安·德赫斯特（Ryan Dewhurst），泰瑞辛·德米特里（Tereshin Dmitry），克里斯蒂安·东（Christian Dong），奥普拉·埃戈尔（Bry Gardiner） ，亨利·胡（Henry Hu），舍尔·兰格坎珀（Sjoerd Langkemper），维尼修斯·亨利克·马兰戈尼（ViníciusHenrique Marangoni），马丁·马西卡诺（Roberto Martelloni），加尔·马克西姆（Gall Maxim），冈田里奥塔罗·冈田（Riotaro Okada），阿比纳夫·塞普帕尔（Afhinav Sejpal），史蒂芬·塞伊斯（Stefaan Seys），尤格什·夏尔马（Yogesh Sharma），普拉汉（Prabhant Singh），斯文·史莱尔（Sven Schleier），尼基·索尼（Nanhil Soni），安南·斯里瓦斯塔库（Anant Shrivastava），弗朗切斯科·斯蒂瓦拉托（Francesco Stillavato） Abdessamad Temmar，Koki Takeyama，Celnokov Vladislav，Leo Wang |
+
+<br><br>
+
+| 语言 |翻译和审核 |
+| --- | --- |
+| 中文 | Peter Chi, and Lex Chien, Henry Hu, Leo Wang |
```
Comment (jack-smith-dh) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
| 繁体中文 | Peter Chi, and Lex Chien, Henry Hu, Leo Wang |
```

**Pull request merged**